homeassistant/components/keyboard_remote/__init__.py

```diff
@@ -36,9 +41,16 @@
                     {
                         vol.Exclusive(DEVICE_DESCRIPTOR, DEVICE_ID_GROUP): cv.string,
                         vol.Exclusive(DEVICE_NAME, DEVICE_ID_GROUP): cv.string,
```

Comment:
Should we require at least one of these exclusive keys or is there value in keeping them optional?

We have a helper `cv.has_at_least_one_key`.